debezium-core/src/test/java/io/debezium/transforms/outbox/EventRouterTest.java

```diff
@@ -711,7 +711,7 @@ public void canSetPartitionWithAdditionalFields() {
     public void shouldFailOnInvalidConfigurationForTopicRegex() {
         final EventRouter<SourceRecord> router = new EventRouter<>();
         final Map<String, String> config = new HashMap<>();
-        config.put(EventRouterConfigDefinition.ROUTE_TOPIC_REGEX.name(), " [[a-z]");
+        config.put(EventRouterConfigDefinition.ROUTE_TOPIC_REGEX.name(), " [[a-z]**");
```

Comment:
If the input regexes need a change, are users aware about it and what exactly would be the change in regex that they need to make for it to be accepted by re2j ?

Reply:
Most of the common Java regexes are compatible with Re2j syntax.
The cases that differ are around complex regexes such as capture groups requiring additional `P` with the group name etc.
Anyway, we have a validator to catch the invalid regexes and show it to the user. 
We haven't seen a case till now where the customers are facing any issue with the regex syntax.